xml/System.Activities/ActivityInstanceState.xml

```diff
@@ -44,15 +44,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>Specifies that the activity is in a canceled state.</summary>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- Canceled is a terminal state for an activity instance. A Canceled activity completed without performing the function it was designed to perform.  
-  
- ]]></format>
-        </remarks>
+        <summary>Specifies that the activity is in a canceled state.<br/>Canceled is a terminal state for an activity instance. A Canceled activity completes without performing the function it was designed to perform. </summary>
```

Comment:
Similar comment as the PR yesterday. Shouldn't we use `<para>` here instead of `<br/>` in these cases?